jenkins-pipelines/upgrade-major-scylla-k8s-eks.jenkinsfile

```diff
@@ -4,6 +4,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingOperatorUpgradePipeline(
     backend: 'k8s-eks',
+    aws_region: 'eu-north-1',
```

Comment:
Cannot we set it in the "defaults/k8s_eks_config.yaml"?

Reply:
Short answer is no.

Long one:
This parameter is consumed by groovy function that is building pipeline, not possible to start hydra at this point.